internal/extproc/translator/gemini_helper.go

```diff
@@ -623,7 +629,55 @@ func geminiLogprobsToOpenAILogprobs(logprobsResult genai.LogprobsResult) openai.
 	}
 }
 
-func buildGCPModelPathSuffix(publisher, model, gcpMethod string) string {
+func buildGCPModelPathSuffix(publisher, model, gcpMethod string, pathParams map[string]string) string {
```

Comment:
why do you need to use map for this? only possible content is `map[string]string{"alt": "sse"}` and i don't see the necessity to use map (causing allocation?). Also, it's converted to a list immediately here and what's the reason for map in the first place. can you just accept variadic string?
```suggestion
// buildGCPModelPathSuffix constructs a path Suffix with an optional queryParams where each string is in the form of "%s=%s".
func buildGCPModelPathSuffix(publisher, model, gcpMethod string, queryParams ...string) string {
```